plugins/outputs/warp10/warp10.go

```diff
@@ -174,7 +174,8 @@ func buildTags(tags []*telegraf.Tag) []string {
 	tagsString := make([]string, len(tags)+1)
 	indexSource := 0
 	for index, tag := range tags {
-		tagsString[index] = fmt.Sprintf("%s=%s", tag.Key, tag.Value)
+		value := strings.Replace(tag.Value, ",", "%2C", -1)
+		tagsString[index] = fmt.Sprintf("%s=%s", tag.Key, value)
```

Comment:
What do you think about query escaping this with the same function used for URL encoding, so all special characters are handled? you could do this for the tag key as well.
```suggestion
		value := url.QueryEscape(tag.Value)
		tagsString[index] = fmt.Sprintf("%s=%s", tag.Key, value)
```

Reply:
Mh, it might be better indeed. After looking again at the Warp10 specification, it seems that both tag key and value should be url encoded, I just didn't encounter an issue with it. But at the same time, most of the characters would be fine, only `,`, `{` and `}` would be a real issue.

I think I'll adapt your commit to url encode the key too, would that be okay? Or I can commit your changes if you prefer to do it.

Reply:
I guess both is fine for @ssoroka. Will remove the "ready-to-merge" label and wait for your changes... :-) Thanks for taking care of this!

Reply:
Alright, I'll make the changes withing a few days, thanks :)